### PR TITLE
feat(transform-prettier): Add transform-prettier

### PR DESCRIPTION
**Pull request by infiniteluke**
**What**:
Add prettier transform

**Why**:
Further ensures consistent and organized readmes.

**How**:
Prettier now supports markdown, including code blocks. 
_NOTE:_ I've used the master branch of prettier here since 1.8 has not released.

**Checklist**:

- [X] Documentation
- [X] Tests
- [X] Ready to be merged 
- [X] Added myself to AUTHORS 

## Notes
* I'm not a huge fan of how it added newlines... But I guess it makes sense and it looks normal: https://github.com/emdaer/emdaer/blob/1edbe49815fd9df6509d51c0bf4ec0eb6974eb3d/README.md
- [x] Also, we shouldn't ship this until it uses a stable version of prettier. 1.8 is planned soon, so I can update this PR once it's shipped. See prettier/prettier#3117 - This is shipped now 🎉

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,10 +83,9 @@ Add a `.emdaer/README.emdaer.md` file:
 
 ```md
 # <!--emdaer-p
-  - '@emdaer/plugin-value-from-package'
-  - value: name
--->
 
+* '@emdaer/plugin-value-from-package'
+* value: name -->
```
Comment (infiniteluke) [inline README.md:102]: This is problematic...

Comment (infiniteluke) [reply README.md:102]: `<!-- prettier-ignore -->` can take care of this since its an edge case. However, it may be worth opening an issue in prettier.

**Comment by infiniteluke** [comment]
This is ready for review again

**Comment by flipactual** [comment]
I'll let you decide how to proceed @infiniteluke – I'm okay with whatever you think works best

**Pull request merged**